pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PauseStatus.java

```diff
@@ -22,20 +22,27 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Set;
+import org.apache.pinot.spi.config.table.TablePauseStatus;
 
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class PauseStatus {
   private boolean _pauseFlag;
   private Set<String> _consumingSegments;
+  private TablePauseStatus.ReasonCode _reasonCode;
+  private String _comment;
   private String _description;
```

Comment:
Let's come to an agreement on the fields:
1. `comment` and `description` are essentially the same thing. Let's just have one (I prefer `comment`).
2. `time` field is missing.
3. Let's remove `consumingSegments`. As of now we don't need it. Also, although for streams with low number of partitions it might be ok to have it, but for large streams with hundreds (or maybe thousands) of partitions, it'll be problematic.

Reply:
1. If either works, can we keep `description` in order to not break any existing usages?
2. Will add
3. Does it make sense to return the consuming segment count as that would be a useful info to track the pause/unpause progress (0 or total # partitions indicating respective operation completion) on the table?

Reply:
1. What existing use case?! PauseStatus is a brand new field.
3. Let's not complicate it, and remove it. We already have a pauseStatus endpoint that is useful for progress report.

Reply:
`PauseStatus` is the API response for the `/tables/{tableName}/pauseStatus` endpoint, not brand new, and I think we should add the `numConsumingSegments` to the payload as a progress indicator.